### DISCUSSION TITLE
Json errors when building code

### DISCUSSION THREAD

**Discussion by Slistack25** [StefanoTesla/TeslaBoard#1] [Q&A] [answered]
Hey I love your project but ive tried every version of your code and i keep getting these errors when i Build it.

Any help would be appreciated! 
![vscode error](https://github.com/StefanoTesla/Ascom-Alpacha-ESP32-Switch-Borad-Dome-Board/assets/141672435/2245e163-8f8d-4154-8630-6ac1b7fe280e)

**Comment by StefanoTesla** [comment, marked as answer]
I update the library, anyway arduinojson7 is compatible with arduinojson6, I correct the deprecation warning, only OTA deprecation still remain untill I can test whit a different esp32 (my remote is 75km from me :D)
